### PR TITLE
[#IC-303] Add Message View to User Data Extract and Delete

### PR DESCRIPTION
**Pull request by AleDore**
#### List of Changes

- Add Message View delete on DeleteUserDataActivity
- Add Message View data on ExtractUserDataActivity

#### Motivation and Context

This PR is needed by legal behaviour in order to manage new `message-view` collection, introduced by pagination.
#### How Has This Been Tested?

Unit tests
#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by CaraDune — APPROVED** [review]
perfect, thanks guys!

**Comment by fabriziopapi** [comment]
I'm not sure if the "extract" part is strictly required.

**Review by fabriziopapi — APPROVED** [review]

File: DeleteUserDataActivity/__tests__/backupAndDelete.test.ts [hunk 1]
```diff
@@ -232,6 +248,30 @@ describe(`backupAndDeleteAllUserData`, () => {
...
+      profileModel,
+      userDataBackup,
+      servicePreferencesModel,
+      fiscalCode: aFiscalCode
+    })();
+
+    expect(E.isRight(result)).toBe(true);
+  });
+
+  it("should stop if there is an error while looking for a message View (404)", async () => {
```
Comment (gquadrati) [inline DeleteUserDataActivity/__tests__/backupAndDelete.test.ts:263]: Add 

`    expect(mockDeleteMessageView).not.toHaveBeenCalled();`

**Review by gquadrati — APPROVED** [review]
Just few comments

**Pull request merged**